code/modules/mob/living/carbon/human/species.dm

```diff
@@ -719,6 +719,7 @@
 	eyes = "blank_eyes"
 	speech_verb_override = "transmits"
 	count_human = TRUE
+	total_health = 80
```

Comment:
Perhaps it may be wise to put in the PR body and changelog that you're decreasing the max health of sectoids in exchange for their psionics

Reply:
I don't remember half of what I even did in this pr tbh